src/modules/previewpane/common/Utilities/SvgPreviewHandlerHelper.cs

```diff
@@ -154,6 +155,34 @@ private static string CheckUnit(string length)
             return length + "px";
         }
 
+        /// <summary>
+        /// Swaps positions of default and svg namespace definitions if default namespace comes first in original SVG data
+        /// </summary>
+        /// <param name="svgData">SVG data</param>
+        /// <returns>Returns modified SVG data</returns>
+        public static string SwapNamespaces(string svgData)
+        {
+            const string defaultNamespacePrefix = "xmlns=\"";
```

Comment:
Wonder if this can cause issues on some cases we're not aware. It'd be safer to look for the bigger strings, since it's always going to be the same, right? `xmlns="http://www.w3.org/2000/svg"` and `xmlns:svg="http://www.w3.org/2000/svg"`

Reply:
I'm not really sure that it will always be the same.. someone might have their own svg implementation I guess :)

We can go for full paths now, and go back to this if we see issues